src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java

```diff
@@ -30,6 +35,8 @@
 @RequiredArgsConstructor
 public class ResourceModificationEventListener {
 
+  private static final Set<ResourceTypeDictionary> AUTHORITY_TYPES = Set.of(CONCEPT, PERSON, FAMILY);
```

Comment:
Instead of defining AUTHORITY_TYPES, it may be better to define `BIBLIOGRAPHIC_TYPES` with 2 values - `INSTANCE` and `WORK`, and change the method in line 95 to `isBibliographic`.

`AUTHORITY_TYPES` will contain a lot of values - CONCEPT, PERSON, FAMILY, JURISDICTION, PLACE, MEETING, TOPIC, TITLE, FORM, TEMPORAL, ....

Reply:
discussed, upgraded